src/components/comment/newComment/CommentForm.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState, useEffect } from 'react';
+
+// MUI
+import Button from '@material-ui/core/Button';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+
+// Redux
+import { connect } from 'react-redux';
+import { getCommentPost } from '../../../redux/actions/dataActions';
+
+const CommentForm = ({ classes = {}, postId, UI, getCommentPost }) => {
+  const [body, setBody] = useState('');
+
+  const handleChange = event => {
+    const { value } = event.target;
+    setBody(value);
```

Comment:
you could oneline this as event.target.value

Reply:
fixed